### PR TITLE
Feature/simpleusecase maestrowf

### PR DESCRIPTION
**Pull request by pdiercks**
Hi @dglaeser, Hi @joergfunger ,
this is an initial version of the simple use case implementation with [maestrowf](https://github.com/LLNL/maestrowf) In current state every process is run locally.
I am unsure how to proceed, i.e. if we want to improve the implementation (run on a HPC machine) since there are still many questions not answered by reading the [documentation](https://maestrowf.readthedocs.io/en/latest/).
For example:
    - how input files are handled (copied to the remote machine)?
    - how results may be retrieved?
    - what possible values can be defined in the `env` block?
Let me know what you think.
Best,
Philipp

### CODE REVIEW

File: simple_use_case/maestrowf/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Maestrowf
+Implementation of the simple use case with [maestrowf](https://github.com/LLNL/maestrowf).
+
+## Installation
+`maestrowf` itself can be installed via pip. It does not deploy the software stack for you, so
+installing all required software first is required. You can do so by running the following commands.
```
Comment (dglaeser) [inline simple_use_case/maestrowf/README.md:6]:
```suggestion
installing all required software first is required. If you have [conda](https://docs.conda.io/en/latest/) set up and running on your system, you can do so by running the following commands.
```

**Comment by dglaeser** [comment]
Thanks @pdiercks, looks good! Good point, so far we have not thought about remote execution, but if a tool supports that out-of-the-box, we should probably try to use it? Or is that interesting rather for the "big" case that we realize afterwards?

**Comment by pdiercks** [comment]
Hi @dglaeser , I would say it is interesting for both cases. However, at the moment I don't know how I can accomplish that with maestrowf and this information seems to be missing in the documentation (at least I did not find that information).

**Comment by pdiercks** [comment]
Hi @dglaeser , I would merge this for now and refactor later if we decide that we would like to make use of remote execution. If we like to include this in the actions probably the `paper.pdf` should be written to the working directory for the upload? (similar to the snakemake case)

**Comment by dglaeser** [comment]
@pdiercks, yes, let's merge this. I'm fine with writing the .pdf wherever, it just needs to be defined for actions to be implementable.  Working directory sounds good to me!

**Pull request merged**